.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+**Description of change**
+
+
+**How to test**
+
+
+**Issue(s)**
+Closes https://github.com/HHS/Head-Start-TTADP/issues/0
+
+**Checklist**
+<!-- Add details to each completed item -->
+- [ ] Meets issue criteria
+- [ ] Code tested
+- [ ] Documentation updated (e.g. Readme, infrastructure and data diagrams, API docs)
+- [ ] Meets accessibility standards
```

Comment:
I love having a template defined. Can we sync this checklist to the checklist on https://github.com/HHS/Head-Start-TTADP/wiki/Definition-of-Done?

Reply:
I looked at the Definition of Done to write these, and thought this was a sufficient summary. Can you give me more detail on what you think is missing?

I ignored CI pipeline, no linting errors, and security scans passed because we'll set it up so you can't merge a PR without the build passing and those things will be part of the build passing.  I left out Deploy to staging, bc deploying isn't a precursor to merge. And the OSCAL templates didn't seem to be something we'd change on a per PR basis. Though, I'm not totally clear on what's being defined by those so maybe I have that wrong.

Reply:
You make an excellent point about the differences between a checklist of things to do before merge (and which of those require any sort of manual review), and when a story can be accepted as done according to the full QASP.

So, I'm good with keeping this checklist mostly as it is, though I think it would be good to replace the generic documentation item with the broken down version of ADR/diagrams/api methods/OSCAL. Most PRs won't change all of them, but I want it to be a very conscious choice each time whether we need to update each of those documentation pieces with that PR.

Reply:
Sounds great- I'll make those changes. 👍

Reply:
https://github.com/adhocteam/Head-Start-TTADP/pull/11